C/scripts/generate_export_list.py

```diff
@@ -0,0 +1,85 @@
+import os
```

Comment:
is this not used now?

Reply:
It's used several times in the script (any time you see a line that uses `os`)

Reply:
sorry, i mean whole script. `generate_export_list.py`

Reply:
It is used to keep the three export files in sync.  The new process should be edit c4.txt, then run the python script to generate the three export lists.  This eliminates forgetting to update one or more of them.

Reply:
whenever we add a new file to lite-core, we will adding them to c4.txt, and run the script `generate_export_list.py`.

Reply:
Not a new file, necessarily, but when you want to add an exported function.  Only functions in this list can be used by consumers of the shared LiteCore library, the rest are hidden.